pkg/cmd/pulumi/testdata/policy/__main__.py

```diff
@@ -0,0 +1,29 @@
+from pulumi_policy import (
```

Comment:
Don't think we actually _need_ anything real here? Can this just be a faked out .txt file and PulumiPolicy.yaml with something like "runtime: mock"?

Reply:
Ah indeed, that seems to work!  Earlier it was complaining that it didn't have the dependencies downloaded, hence me adding a proper policy, but with `runtime: mock` it seems to just work.  Much nicer, thanks!